src/components/Homepage.js

```diff
@@ -1,9 +1,84 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import '../styles/Home.scss';
+import Loading from './Loading';
+import Jeep from '../assets/images/7vylxq_large.png';
+import { SampleNextArrow, SamplePrevArrow } from './arrows';
 
-const Homepage = () => (
-  <section>
-    <h1>Homepage</h1>
-  </section>
-);
+const Homepage = () => {
+  const cars = useSelector((state) => state.reducers.cars);
+
+  const settings = {
+    className: 'center',
+    centerMode: true,
+    infinite: true,
+    centerPadding: '100px',
+    dots: true,
+    slidesToShow: 3,
+    slidesToScroll: 2,
+    lazyLoad: true,
+    nextArrow: <SampleNextArrow />,
+    prevArrow: <SamplePrevArrow />,
+    initialSlide: 0,
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          slidesToShow: 3,
+          slidesToScroll: 1,
+          infinite: true,
+          dots: true,
+        },
+      },
+      {
+        breakpoint: 600,
+        settings: {
+          slidesToShow: 2,
+          slidesToScroll: 2,
+          initialSlide: 2,
+        },
+      },
+      {
+        breakpoint: 480,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1,
+          nextArrow: false,
+          prevArrow: false,
+        },
+      },
+    ],
+  };
+
+  return (
+    <div className="home-container">
+      <h1>Available Cars</h1>
+      <h5>Check our best available cars</h5>
+      <div className="cards-container caroussel">
+        <Slider {...settings}>
+          {!cars.length ? <Loading /> : (
+            cars.map((car) => (
+              <Link to={`car/${car.name}`} key={car.id}>
+                <div className="car-card">
+                  <img src={Jeep} alt={car.name} />
+                  <div className="descript">
+                    <h2>
+                      {car.name.slice(0, 9)}
+                    </h2>
+                  </div>
+                </div>
+              </Link>
+            ))
+          )}
+        </Slider>
```

Comment:
### There are some issues with the design of your carousel. See below

# 1. Mobile Version

![mobile (copy)](https://user-images.githubusercontent.com/93039359/175781336-9b5367b1-28e7-4668-bfea-9f8a5cd863d5.png)

- [ ] In the yellow box, you can see each item is overlapping the previous item. Please fix it. In the mobile version, there should be one item visible not 2 or 3.

# 2. Desktop Version
![desktop carousel (copy)](https://user-images.githubusercontent.com/93039359/175781683-58414654-67e0-43c9-acdf-8433e4460bdd.png)

- [ ] In the yellow box you can see your carousel is overlapping the navbar. 
Please make your carousel `z-index: -1;` - 
- [ ] Navigation bar has no background so please make it `background-color: white;`
- [ ] You can see in the red box that your flicker which is responsible for moving the carousel items. It hides behind the navbar when you change the resolution of the screen. Please fix it. **(You will see this change after making your background-color: white; for the navbar)**